oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/generated/endpoint/EndpointAvgIndicator.java

```diff
@@ -37,10 +38,10 @@
  */
 @IndicatorType
 @StreamData
-@StorageEntity(name = "endpoint_avg", builder = EndpointAvgIndicator.Builder.class)
+@StorageEntity(name = "endpoint_avg", builder = EndpointAvgIndicator.Builder.class, source = Scope.Endpoint)
 public class EndpointAvgIndicator extends LongAvgIndicator implements AlarmSupported {
 
-    @Setter @Getter @Column(columnName = "entity_id") @IDColumn private String entityId;
+    @Setter @Getter @Column(columnName = "entity_id") @IDColumn private java.lang.String entityId;
```

Comment:
Why import the full package for the String object?

Reply:
The exists oal tool did this. Not my change for this.

Reply:
Look like all classes are using full package.